src/main/java/bio/terra/cli/businessobject/Workspace.java

```diff
@@ -429,12 +420,4 @@ public String getUserEmail() {
   public List<Resource> getResources() {
     return Collections.unmodifiableList(resources);
   }
-
-  public OffsetDateTime getCreatedDate() {
-    return createdDate;
-  }
-
-  public OffsetDateTime getLastUpdatedDate() {
-    return lastUpdatedDate;
-  }
```

Comment:
These seem relevant even for non data collection workspaces, is it worth keeping them?